protostar/testing/environments/test_execution_environment.py

```diff
@@ -18,6 +18,7 @@
 
 from .common_test_cheatcode_factory import CommonTestCheatcodeFactory
 from .execution_environment import ExecutionEnvironment
+from ...compiler import ProjectCompiler
```

Comment:
nitpick: wouldn't it be easier to import `protostar.compiler`?

Reply:
it's just the thing that IDE proposed, since it saw more relative imports. will fix.

Reply:
done